pennylane/numpy/tensor.py

```diff
@@ -154,6 +153,9 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         # of the vectorized ufunc
         res = super().__array_ufunc__(ufunc, method, *args, **kwargs)
 
+        if isinstance(res, Operator):

```

Comment:
I was just thing about this.  Maybe if we don't want this code to be dependent on `Operator` and we want it to be open to extension:
```suggestion
        if not isinstance(res, _np.ndarray):
```

Reply:
Ignore that doesn't work ;(